src/main/jte/mail/booking_deleted.jte

```diff
@@ -3,17 +3,26 @@
 @import edu.kit.hci.soli.dto.BookingDeleteReason
 
 @param JteContext context
+@param String css
 
 @param Booking booking
 @param BookingDeleteReason reason
 
-<%--TODO: localize!--%>
-<p>
-    Dear ${booking.getUser().getUsername()},
-</p>
-<p>
-    unfortunately, your booking for the room ${booking.getRoom().getId()} starting at ${context.format(booking.getStartDate())} has been deleted.
-</p>
-<p>
-    Reason: ${context.localize("bookings.delete_reason." + reason)}
-</p>
\ No newline at end of file
+@template.mail.layout(title= context.lookup("mail.booking_deleted.heading"), content = @`
+    !{var deleteReason = context.localize("bookings.delete_reason." + reason);}
+    <style>
+        ${css}
+    </style>
+
+    <p>
+        ${context.localize("mail.booking_deleted.message", booking.getUser().getUsername(), booking.getRoom().getId().toString(), booking.getStartDate().toString())}<br>
+        ${deleteReason}<br><br>
+        ${context.localize("mail.booking_deleted.last_phrase")}<br>
+        ${context.localize("mail.ending")}<br><br>
+    </p>
+    <div class="btn btn-accent">
+        <a class="pretty-link" href= "${context.getHostname()}${booking.getRoom().getId()}">
```

Comment:
Ideally, the link should wrap the div, not the other way around
I'm also not entirely sold on having a paragraph inside a button.